### DISCUSSION TITLE
Seeed XIAO BLE Sense

### DISCUSSION THREAD

**Discussion by kise1-off** [edge-ml/edge-ml#27] [Q&A]
Hello!
Do you have plans for providing support for Seeed XIAO BLE Sense?

**Comment by TobiasRoeddiger** [comment]
We didn‘t have the chance to get our hands on a device yet. I‘ll see what we can do.

**Comment by TobiasRoeddiger** [comment]
FYI: we will release support for Xiao BLE Sense within next week.
